pages/app-deploy/laravel/email.js

```diff
@@ -19,7 +19,7 @@ export default () => (
       </div>
     </div>
 
-    <h3>ارسال ایمیل</h3>
+    <h3>ارسال ایمیل در لاراول 10</h3>
```

Comment:
We usually use Persian numeric format:

```suggestion
    <h3>ارسال ایمیل در لاراول ۱۰</h3>
```